layouts/_default/single.html

```diff
@@ -30,6 +30,7 @@ <h1 class="post-title">
   <div class="post-content">
     {{ .Content }}
   </div>
+  {{ if not .Site.Params.disableReadOthers }}
   {{ if or .NextInSection .PrevInSection }}
```

Comment:
If you wrap this in another condition, please indent this block for better readability ;-)